samples/bluetooth/direction_finding_connectionless_rx/README.rst

```diff
@@ -112,6 +112,43 @@ The following table presents the patterns that you can use to switch antennas on
 | RFU    | 15 (0b1111)  |
 +--------+--------------+
 
+Constant Tone Extension transmit and receive parameters
+=======================================================
+
+Constant Tone Extension works in one of two modes: angle of arrival (AoA) or angle of departure (AoD).
+The transmitter is configured with operation mode that is used for CTE transmission.
+The receiver operating mode depends on configuration, by default both AoA and AoD modes are enabled, and CTE type received in CTEInfo filed in PDUs extended advertising header.
+Depending on operation mode selected for reception of CTE, the receivers radio peripheral will execute antenna switching and CTE sampling (AoA) or CTE sampling only (AoA).
+
+There are two antenna switch slot length allowed: 1 us or 2 us.
+The transmitter uses the antenna switch slot length to configure radio peripheral in Angle of Deprature mode only.
+The receiver uses the local setting of antenna switch slot length in Angle of Arrival mode only.
+When CTE type in received PDU is AoD, the receivers radio peripheral uses the antenna switch slot length appropriate for AoD type: 1 us or 2 us.
+The antenna switch slot length has influece on number of IQ samples provided in IQ samples report.
+
+Constant Tone Extension length is limited to range 16 us to 160 us, the value is provided in 8 us units.
+The transmitter is responsible for setting of CTE length.
+The value is send to receiver as part of CTEInfo field in PDUs extended advertising header.
+The receivers radio peripheral uses the CTE length, provided in periodic advertising PDU, to execute antenna switching and CTE sampling in AoA mode or CTE sampling only in AoD mode.
+The CTE length has influence on number of IQ samples provided in IQ samples report.
+
+Constant Tone Extension consists of three periods:
+
+* Guard period that lasts for 4 us. It is a gap before actual CTE reception and adjacent PDU transmission to avoid interference.
+* Reference period where single antenna is used sampling, samples are spaced each other by 1 us. The period duration is 8 us.
+* Switch-sample period that is split into switch and sample slots. Every slot may either be 1 us or 2 us long.
+
+Total number of IQ samples provided by IQ samples report varies.
+It depends on the CTE length and the antenna switch slot length.
+There will always be 8 samples from reference period, 1 to 37 samples with 2 us atnenna switching slots, 2 to 74 samples with 1 us antenna switching slots, meaning 9 to 82 samples in total.
+
+For example CTE length is 120 us, antenna switching slot is 1 us.
+There are 8 samples from reference period.
+Switch sample period duration is: 120 us - 16 us = 104 us.
+For antenna switching slot 1 us, sample is taken every 2 us (1 us antenna switch slot, 1 us sample slot).
+Number of samples from the switch-sample period is 104 us / 2 us = 52.
+Total numnber of samples is 60.
```

Comment:
```suggestion
This will result in the following number of IQ samples:

* 8 samples from the reference period.
* Switch sample period duration is: 120 µs - 16 µs = 104 µs. For the 1 µs antenna switching slot, the sample is taken every 2 µs (1 µs antenna switch slot, 1 µs sample slot). Meaning, the number of samples from the switch-sample period is 104 µs / 2 µs = 52.

The total number of samples is 60.
```